common/src/main/resources/META-INF/openapi.json

```diff
@@ -3866,6 +3990,93 @@
                 "example": {
                     "owner": "bwayne"
                 }
+            },
+            "GroupMetaData": {
+                "title": "Root Type for GroupMetaData",
+                "description": "",
+                "required": [
+                    "id",
+                    "description",
+                    "artifactsType",
+                    "createdBy",
+                    "createdOn",
+                    "modifiedBy",
+                    "modifiedOn",
+                    "properties"
+                ],
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "$ref": "#/components/schemas/GroupId",
+                        "description": ""
+                    },
+                    "description": {
+                        "type": "string"
+                    },
+                    "type": {
+                        "$ref": "#/components/schemas/ArtifactType",
+                        "description": ""
+                    },
```

Comment:
I think for the Apicurio V2 API we should not include this.  I don't think we want to imply that this will be enforced in any way.  We don't want groups to limit what kinds of artifacts they can have.  At least I don't think so.  Certainly not at this point in the dev cycle (release pending soon).

Reply:
Ok, but note that is is part of the SQL model already, I was just following that model.

Reply:
And, actually, I think this is just "metadata", and is not being actually enforced.

Reply:
Good that it's not enforced.  But I also think we should not expose it via the v2 API because it will **imply** that it will be enforced.  Is it OK for the value to be `NULL`?

Reply:
Yes, it's nullable, I'm not setting the type in most of the tests. Actually, I don't think the second part is true. Even if it has a value, it will be used just as metadata of the group, it won't be enforced in any way (at least that I'm aware).